src/express/middlewares/img-storage.js

```diff
@@ -0,0 +1,19 @@
+const multer = require(`multer`);
+const path = require(`path`);
+const {nanoid} = require(`nanoid`);
+
+const UPLOAD_DIR = `../upload/img/`;
+
+const uploadDirAbsolute = path.resolve(__dirname, UPLOAD_DIR);
+
+const storage = multer.diskStorage({
+  destination: uploadDirAbsolute,
```

Comment:
```suggestion
const storage = multer.diskStorage({
  destination: path.resolve(__dirname, UPLOAD_DIR),
```
как опция - можно сократить, поскольку такая запись тоже считается интуитивно понятной